enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1334 @@
+    status: "True"  
+    message: "Details on what..."
+```
+
+#### Change Management Metrics
+Change management information will be made available through cluster metrics. Each resource
+containing the `spec.changeManagement` stanza must expose the following metrics:
+- Whether any change management strategy is enabled.
+- Which change management strategy is enabled. This can be used to notify SRE when a cluster begins using a 
+  non-standard strategy (e.g. during emergency corrective action).
+- The number of seconds until the next known permitted change window. See `change_management_next_change_eta` metric. 
+  This might be used to notify an SRE team of an approaching permissive window.
+- The number of seconds until the current change window closes. See `change_management_permissive_remaining` metric.
+- The last datetime at which changes were permitted (can be nil). See `change_management_last_change` metric (which 
+  represents this as seconds instead of a datetime). This could be used to notify an SRE team if a cluster has not 
+  had the opportunity to update for a non-compliant period.
+- If changes are pending due to change management controls. When combined with other 
+  metrics (`change_management_next_change_eta`, `change_management_permissive_remaining`), this can be used to 
+  notify SRE when an upcoming permissive window is going to initiate changes and whether changes are still 
+  pending as a permissive window closes.
+
+### Enhanced MachineConfigPool Control
+The MachineConfigOperator (MCO), like any other operator, works toward eventual consistency of the system state
+with state of configured resources. This behavior offers a simple user experience wherein an administrator
+can make a change to a `MachineConfig` and the MCO take over rolling out that change. It will
+1. Aggregate selected `MachineConfig` objects into a new "rendered" `MachineConfig` object.
+1. Work through updating nodes to use the "latest" rendered `MachineConfig` associated with them. 
+
+However, in business critical clusters, progress towards the "latest" rendered `MachineConfig` offers less control
+than may be desired for some use cases. To address this, the `MachineConfigPool` resource will be extended
+to include an option to declare which rendered `MachineConfig` the MachineConfigOperator should make
+progress toward instead of the "latest".
+
+```yaml
+kind: MachineConfigPool
+spec:
+  machineConfig:
+    # An administrator can identify the exact rendered MachineConfig
+    # the MCO should progress towards for this MCP.
+    name: <rendered machine configuration name>
+    
+    # Validation defines when MCO will allow the use of a new
+    # configuration. Default means a node must successfully use the
+    # configuration before new nodes are ignited with the
+    # desiredConfiguration.
+    # None means no validation is necessary (new nodes will
+    # immediately ignite with the configuration).
+    validation: Default

```

Comment:
Feel like we can come up with better names than None and Default, they aren't particularly descriptive to an end user.

Maybe something like UpgradeFirst, or CanaryUpgrade 🤔